### ISSUE TITLE
Staff portait on staff specialists page is missing alternative text

### ISSUE THREAD

**Issue by axamei** [pulibrary/pul_library_drupal#1377]
Error: Img element is the only content of the link, but is missing alt text. The alt text should describe the purpose of the link.
